src/opm/parser/eclipse/EclipseState/Grid/GridProperty.cpp

```diff
@@ -425,11 +433,13 @@ namespace Opm {
 template<>
 void GridProperty<int>::setDataPoint(size_t sourceIdx, size_t targetIdx, const DeckItem& deckItem) {
     m_data[targetIdx] = deckItem.get< int >(sourceIdx);
+    m_defaulted[targetIdx] = false;
 }
 
 template<>
 void GridProperty<double>::setDataPoint(size_t sourceIdx, size_t targetIdx, const DeckItem& deckItem) {
     m_data[targetIdx] = deckItem.getSIDouble(sourceIdx);
+    m_defaulted[targetIdx] = false;
```

Comment:
I think there are quite many additional functions in `GridProperty.cpp` which needs an addition of the `m_defaulted[targetIdx] = true`.